### ISSUE TITLE
location of acme.inc

### ISSUE THREAD

**Issue by onley** [blueslow/sslcertzoneedit#4]
I have poured over the documentation but cannot seem to locate this acme.inc into which I am to insert the zoneedit credentials. I assume this was an include file that appears to have been done away with. Have I missed something or is there possibly a new approach?

**Comment by blueslow** [comment]
Assuming acme  pakage is installed in pfsense. In pfsense the the acme tools is installed under  /usr/local/pkg/acme where acme.inc can be found. To locate acme tools in other systems i sugest using find, e.g. 
`find / -iname acme`
alternatively 
`find / -iname .acme`

**Comment by onley** [comment]
I'm sorry, I wanted to use acme.sh standalone. I just wanted to run it from the cli to generate and renew certs. The installer placed the files in my home directory and there is no acme.inc there or anywhere else I can find.

Thank you for the prompt reply.

**Comment by blueslow** [comment]
I'm sorry the standalone documentation is wrong,i will will update it.
There is no acme.inc in standalone  installation  The acme.inc is only available in pfsense in order  provide fields in the webgui to enter the token et.c. for zone edit.  In standalone, if I remember the token id are provided by environment variables or cli arguments. The important file is in dnsapi specifically dns_zoneedit.sh. The  dns_zoneedit.sh file needs the following 
ZONEEDIT_Token and ZONEEDIT_ID, which should correspond to your "credentials" at Zoneedit

Example:
export ZONEEDIT_Token=yourtoken
export ZONEEDIT_ID=yourid
acme.sh --issue -d example.com --dns dns_zoneedit

For more info,  please these guide 
https://github.com/acmesh-official/acme.sh
https://github.com/acmesh-official/acme.sh/wiki/DNS-API-Dev-Guide

**Comment by onley** [comment]
Thank you. I created simple auth and cleanup scripts and used certbots hooks. I have it working now. I will look into this way too as my approach in not that elegant.

**Comment by blueslow** [comment]
Thank you for notifying me regarding the error in the README.md documentation.

**Comment by onley** [comment]
Works like a charm. Thank You!

**Comment by blueslow** [comment]
Glad to hear.

**Comment by onley** [comment]
I did a little hacking on the script and managed to get it to delete the text records but my coding is not really up to posting. All I did was create a conditional so that if the dns_zoneedit_rm() {  function  calls _zoneedit_api() { the urls get rewritten
`
Zoneedit_API="https://dynamic.zoneedit.com/txt-create.php"
Zoneedit_API_GET="https://%s:%s@dynamic.zoneedit.com/txt-create.php?host=%s&rdata=%s"
if [[ "$mode" == "DELETE" ]]; then
  Zoneedit_API="https://dynamic.zoneedit.com/txt-delete.php"
  Zoneedit_API_GET="https://%s:%s@dynamic.zoneedit.com/txt-delete.php?host=%s&rdata=%s"
fi
`

**Comment by blueslow** [comment]
I see that you changed the txt-create.php to txt-delete.php. I'm not fluent in PHP or rest api and to my knowledge Zoneedit have not provided any delete functionality.  So I don't under stand how it can work, please explain?  

Currently I have hard  to find the time for change and tests, maybe i find the time between Christmas and new years eve.

**Comment by onley** [comment]
I actually took the conditional out, I just moved the global definitions into the add and rm subs. Zoneedit recently added the endpoint to delete txt records and it operates identically to the create with the name delete. Your code works perfectly out of the box all I had to do is define the globals Zoneedit_API and Zoneedit_API_GET in the subs  dns_zoneedit_add() { dns_zoneedit_rm() { . _zoneedit_api() { remains untouched and executes the appropriate endpoint based on what the add() or rm() sends for the url.
